data/cache/.gitignore

```diff
@@ -1,2 +1,2 @@
-*
-!.gitignore
+*
+!.gitignore

```

Comment:
I'm confused -- this file has been restored, but it's still showing as having diffs. I'm not sure why!

Reply:
Hi Demian, just force-pushed after rebase and re-restoring .gitignore -- it has some content now. Pls. let me know if it worked out allright. Cheers!

Reply:
That's sorted it out -- thanks!